triage/forms.py

```diff
@@ -136,6 +145,26 @@ class CompaniesHouseSearchForm(forms.Form):
     term = forms.CharField()
 
 
+class SectorForm(forms.Form):
+    sector = forms.ChoiceField(
+        choices=BLANK_CHOICE_DASH + SECTORS_CHOICES,
+        label='',
+        label_suffix='',
+        widget=forms.Select(attrs={'id': 'js-sector-select'}),
+    )
+
+    def update_triage_sector(self, request, answers):
+        answers.update(self.cleaned_data)
+        answer_manager = TriageAnswersManager(request)
+        answer_manager.persist_answers(answers)
```

Comment:
it's a bit funky performing IO (in this case, saving to the database) inside the form.

I would move the view, where it's more obvious to the reader a side-effect is happening.